<<<<<<< HEAD
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.517, y1:0.539773, x2:1, y2:0, stop:0 rgba(170, 170, 255, 255), stop:1 rgba(255, 85, 255, 255))</string>
   </property>
   <widget class="QPushButton" name="addstudent_bttn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(44, 227, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>AddNewStudent</string>
    </property>
   </widget>
   <widget class="QPushButton" name="financedata_bttn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(44, 227, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>FinancialData</string>
    </property>
   </widget>
   <widget class="QPushButton" name="studentdata_bttn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(44, 227, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>StudentData</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout_bttn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(44, 227, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>LogOut</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
=======
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.517, y1:0.539773, x2:1, y2:0, stop:0 rgba(170, 170, 255, 255), stop:1 rgba(255, 85, 255, 255))</string>
   </property>
   <widget class="QPushButton" name="addstudent_bttn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(44, 227, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>AddNewStudent</string>
    </property>
   </widget>
   <widget class="QPushButton" name="financedata_bttn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(44, 227, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>FinancialData</string>
    </property>
   </widget>
   <widget class="QPushButton" name="studentdata_bttn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(44, 227, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>StudentData</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout_bttn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(44, 227, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>LogOut</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
>>>>>>> parent of 5a30ca7... commit message
